### PR TITLE
Add Apple 2020 iPhone and iPad device info

### PR DESCRIPTION
**Pull request by LisonFan**
add iPhone 12 mini, iPhone 12, iPhone 12 Pro, iPhone 12 Pro Max and iPad Air4 models info.
Adjust the sorting of iPad models info by screen size and release year.

https://www.theiphonewiki.com/wiki/Models

https://www.apple.com/iphone-12-pro/specs/
https://www.apple.com/iphone-12/specs/

https://www.apple.com/ipad-air/specs/ (10/24/2020 page snapshot: https://user-images.githubusercontent.com/9424775/97076422-39cfaa80-1614-11eb-9128-2816dede4734.png)

### CODE REVIEW

File: GBDeviceInfo/GBDeviceInfo_iOS.m [hunk 1]
```diff
@@ -201,6 +201,9 @@ + (NSArray *)_modelNuances {
                 // SE
                 @[@8, @4]: @[@(GBDeviceModeliPhoneSE), @"iPhone SE", @(GBDeviceDisplay4Inch), @326],
                 
+                // SE 2
+                @[@12, @8]: @[@(GBDeviceModeliPhoneSE2), @"iPhone SE 2", @(GBDeviceDisplay4p7Inch), @326],
```
Comment (goncharik) [inline GBDeviceInfo/GBDeviceInfo_iOS.m:205]: Let's keep sorting by device version numbers

File: GBDeviceInfo/GBDeviceInfo_iOS.m [hunk 2]
```diff
@@ -267,11 +274,23 @@ + (NSArray *)_modelNuances {
                 @[@3, @4]: @[@(GBDeviceModeliPad4), @"iPad 4", @(GBDeviceDisplay9p7Inch), @264],
                 @[@3, @5]: @[@(GBDeviceModeliPad4), @"iPad 4", @(GBDeviceDisplay9p7Inch), @264],
                 @[@3, @6]: @[@(GBDeviceModeliPad4), @"iPad 4", @(GBDeviceDisplay9p7Inch), @264],
+                
+                // iPad 5th Gen, 2017
+                @[@6, @11]: @[@(GBDeviceModeliPad5), @"iPad 2017", @(GBDeviceDisplay9p7Inch), @264],
```
Comment (goncharik) [inline GBDeviceInfo/GBDeviceInfo_iOS.m:279]: Let's keep sorting by device version numbers

**Review by goncharik — CHANGES_REQUESTED** [review]

**Comment by goncharik** [comment]
Please add iPad 8th Gen as well - https://www.theiphonewiki.com/wiki/IPad_(8th_generation)

**Comment by lmirosevic** [comment]
Merged #116 instead. 

Thanks to both @goncharik and @LisonFan

**Pull request closed without merging**